apps/spruce/src/pages/waterfall/WaterfallMenu/index.tsx

```diff
@@ -17,6 +18,7 @@ export const WaterfallMenu: React.FC<Props> = ({ projectIdentifier }) => {
   const dropdownItems = [
     <GitCommitSearch key="git-commit-search" setMenuOpen={setMenuOpen} />,
     <JumpToMostRecent key="jump-to-most-recent" setMenuOpen={setMenuOpen} />,
+    <ClearAllFilters key="clear-all-filters" setMenuOpen={setMenuOpen} />,
     <AddNotification
```

Comment:
Thoughts on adding icons to these? We could do like... `Code`, `ClockWithArrow`, `Eraser`, `Megaphone`?

Reply:
how is this:

<img width="253" alt="Screenshot 2025-02-26 at 12 37 12 PM" src="https://github.com/user-attachments/assets/61c69579-0c56-4b01-9260-59fb4560a85b" />

some more options:

<img width="251" alt="Screenshot 2025-02-26 at 12 36 59 PM" src="https://github.com/user-attachments/assets/e1053ffa-e45b-4257-9041-00d820f78624" />

sorry i would ask you on slack but it logged me out and now im locked out for the time being i guess https://slack-status.com/

Reply:
Oh good finds! `NoFilter` and `Bell` are great. I still slightly prefer `ClockWithArrow` over `Return` but your call.

Thanks 😻